best-practices/README.md

```diff
@@ -104,11 +104,18 @@ Rails
 Testing
 -------
 
+(See also the rspec style guide that appears in this section)
+
+* First #describe what you are doing.
+* Then establish the #context.
```

Comment:
So perhaps we should add - and watch out for nested contexts ... this is indicative of a design problem. WDYT?